datafusion/proto/src/physical_plan/mod.rs

```diff
@@ -486,7 +486,7 @@ impl AsExecutionPlan for PhysicalPlanNode {
                     physical_aggr_expr,
                     physical_filter_expr,
                     input,
-                    Arc::new(input_schema.try_into()?),
+                    physical_schema,
```

Comment:
Implicit clone in conversion function, could be replaced by a variable created before.